lng/english.lng.php

```diff
@@ -2092,3 +2092,6 @@
 $lng['serversettings']['phpfpm_settings']['restart_note'] = 'Attention: The config won\'t be checked for any errors. If it contains errors, PHP-FPM might not start again!';
 $lng['serversettings']['phpfpm_settings']['custom_config']['title'] = 'Custom configuration';
 $lng['serversettings']['phpfpm_settings']['custom_config']['description'] = 'Add custom configuration to each PHP-FPM version instance, for example <i>pm.status_path = /status</i> for monitoring. Variables below can be used here. ' . ' <strong>' . $lng['serversettings']['phpfpm_settings']['restart_note'] . '</strong>';
+
+$lng['serversettings']['awstats']['logformat']['title'] = 'LogFormat setting';
+$lng['serversettings']['awstats']['logformat']['description'] = 'If you use customized logformat for your webserver, you need change the awstats LogFormat too.<br/>Default is 1. For more information check documentation <a target="_blank" href="https://awstats.sourceforge.io/docs/awstats_config.html#LogFormat">here</a>. Might also require changes in the awstats config.';
```

Comment:
why would it require changes to the awstats configuration? There is a placeholder which is replaced by froxlor's traffic cron. Only time the config needs adjustment is when updating...maybe we should add a hint for this in the preconfig (I can do that after merging)

Reply:
Then I apparently understood your comment in the other PR wrong. Sorry! I just removed it from the language files and I'm now going over to fix the conflicts.

Beside that: You could also use "Merge & Squash" (or however the option is called exactly) so that it doesn't mess up your clean Git history in the main branch and keeps it clean.